src/jit/lsra.cpp

```diff
@@ -355,6 +355,35 @@ RegRecord* LinearScan::getRegisterRecord(regNumber regNum)
 }
 
 #ifdef DEBUG
+
+//----------------------------------------------------------------------------
+// getConstrainedRegMask: Returns new regMask which is the intersection of
+// regMaskActual and regMaskConstraint if the new regMask has at least 
+// minRegCount registers, otherwise returns regMaskActual.
```

Comment:
Why wouldn't you simply reduce the set to have minRegCount registers? That would still provide some "stress value" over defaulting to the full candidate set.